pkg/cri/resource-manager/cache/affinity.go

```diff
@@ -341,3 +347,17 @@ func GlobalAffinity(key string, weight int32) *Affinity {
 func GlobalAntiAffinity(key string, weight int32) *Affinity {
 	return GlobalAffinity(key, -weight)
 }
+
+// AddImplicitAffinities registers a set of implicit affinities.
+func (cch *cache) AddImplicitAffinities(implicit map[string]*ImplicitAffinity) error {
+	for name := range implicit {
+		if existing, ok := cch.implicit[name]; ok {
+			return cacheError("implicit affinity %s already defined (%s)",
+				name, existing.Affinity.String())
```

Comment:
Is an error or just overriding the previous rule "the right thing to do"? I don't know. But what if a policy would like to change the implicit affinity rules. for example?

Reply:
That is true and I was considering adding a DeleteImplicitAffinities(name string), but then I didn't bother. Mostly because we don't need it ATM, but also because, since we have the Eligible callback/function in ImplicitAffinity, you can technically always use that work around the lack of Delete if you need to dynamically change the per-container affinity behavior for your policy. I can add the Delete functions nevertheless...

Reply:
Added cache.DeleteImplicitAffinities([]string).